test/integration/timesync_test.go

```diff
@@ -0,0 +1,53 @@
+package test_test
+
+import (
+	"os/exec"
+	"strconv"
+
+	// "github.com/crc-org/crc/test/extended/crc/cmd".
+	"github.com/crc-org/crc/test/extended/crc/cmd"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	sleepSeconds         = "30"
+	maxDateDiffInSeconds = 10
+)
+
+var _ = Describe("Guest machine time should be in sync with host", Label("guest-timesync", "darwin"), func() {
+
+	BeforeSuite(func() {
+		It("Ensuring instance is running")
+		if err := cmd.CheckCRCStatus("running"); err != nil {
+			// cluster is not running
+			startInstance()
+		}
+	})
+
+	It("checks instance date is in sync with host after host suspension", func() {
```

Comment:
I'm not sure what good practice is in Ginkgo, but I imagine `It` blocks to be smaller. One of the reasons for this is that the results file is iterated by `It` blocks. If only one block present, then only one pass/fail result. Which is probably good for e2e test (?) but not sure about integration... More detail in the results might be easier to debug too. But I'm not an expert here, just suggesting!